static/app/views/settings/organizationMembers/organizationMemberDetail.tsx

```diff
@@ -286,6 +286,7 @@ class OrganizationMemberDetail extends DeprecatedAsyncView<Props, State> {
 
     const {access, orgRoleList} = organization;
     const canEdit = access.includes('org:write') && !this.memberDeactivated;
+    const isPartnershipUser = !!member.flags['partnership:restricted'];
```

Comment:
this casts the undefined to false if necessary

Reply:
You can do `member.flags['partnership:restricted'] === true` to be more clear.